README.rst

```diff
@@ -75,27 +75,101 @@ Awesome, isn’t it?
 Development
 ===========
 
-Run the tests
--------------
+You can follow the next steps in order to host postgraas_server locally and be able to develop features or bug fixes:
 
-You need to have docker installed
+Clone repository::
 
-Make sure you pull the right docker image::
+    git clone https://github.com/blue-yonder/postgraas_server
 
-    docker pull postgres:9.4
+*[optional]* Create and run a virtual environment
+
+*This step will help you to not install all needed python packages on your home computer, but in a virtual disposable environment so it doesn't cause any trouble with your local setup*::
```

Comment:
Not sure, _how_ verbose we want to be.
Do others want to chime in on that.
I'd personally omit this from the main README.

The information is good to get people started in Python and GitHub, but if you look at other Python packages, e.g. https://github.com/psf/black or https://github.com/psf/requests the main README **focuses** on using the tool.

Reply:
Given the somewhat convoluted set-up it's probably not wrong to be a little more explicit.

Reply:
Since I'm new to Python and pretty much all technologies used in this project, I tried to create a more step-by-step documentation for future new hires. But I do agree repos should be just focused on the actual tool's usage. It makes more sense to include my findings in a wiki page. I deleted this from the README